taxi/models.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+
+from django.contrib.auth.models import AbstractUser
+
+from django.db import models
+
+
+# Create your models here.
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(
+        Manufacturer,
+        on_delete=models.CASCADE,
+        related_name="cars"
+    )
+    drivers = models.ManyToManyField(
+        settings.AUTH_USER_MODEL,
+        related_name="cars"
+    )
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+    username = models.CharField(max_length=255, unique=True)
```

Comment:
No need to specify the below fields - they are already included in AbstractUser.
Also your Drivers still show as Users on admin panel - please fix that